INSERT INTO d_star (class, a) VALUES ('d', 14);
ERROR:  42P01: relation "d_star" does not exist
LINE 1: INSERT INTO d_star (class, a) VALUES ('d', 14);
                    ^
